src/gsi/statsconv.f90

```diff
@@ -676,18 +678,46 @@ subroutine statsconv(mype,&
         close(iout_wspd10m)
      end if
   end if
+! Summary report for conventional gnssrspd
```

Comment:
The order in which things are done in this block differs from the processing of other obs.  Is there a reason for this?

Reply:
This section follows the order of 2dtm in the same file. I did notice that statsconv.f90 in the develop branch has been updated. I will match the develop branch